extensions/notebook-renderers/src/textHelper.ts

```diff
@@ -54,6 +54,14 @@ function truncatedArrayOfString(id: string, buffer: string[], linesLimit: number
 function scrollableArrayOfString(id: string, buffer: string[], container: HTMLElement, trustHtml: boolean) {
 	const scrollableDiv = document.createElement('div');
 	scrollableDiv.classList.add(scrollableClass);
+	scrollableDiv.onscroll = (e) => {
```

Comment:
Could this be rewritten as a disposable listener?

Reply:
good call, definitely should do that

Reply:
https://github.com/microsoft/vscode/pull/176574